meilisearch-http/src/routes/synonym.rs

```diff
@@ -33,50 +43,57 @@ pub async fn get(ctx: Request<Data>) -> SResult<Response> {
         }
     }
 
-    Ok(tide::Response::new(200).body_json(&synonyms).unwrap())
+    Ok(HttpResponse::Ok().json(synonyms))
```

Comment:
Same here and for every similar places, please don't mixup `web::Json()` and `HttpResponse::Ok().json()`, choose one and use it everywhere. I prefer the `HttpResponse::Ok().json()` because it is consistent with `HttpResponse::Accepted` and others.

Reply:
totally agree. I will change the `web::Json()` to `HttpResponse::Ok().json()` everywhere.